modules/common/test/helpers/configmap.go

```diff
@@ -96,11 +95,11 @@ func (tc *TestHelper) CreateConfigMap(name types.NamespacedName, data map[string
 	return tc.CreateUnstructured(raw)
 }
 
-// AssertConfigMapExists used only in lib-common tests
-func (tc *TestHelper) AssertConfigMapExists(name types.NamespacedName) *corev1.ConfigMap {
+// AssertConfigMapDoesNotExist ensures the ConfigMap resource does not exist in a k8s cluster.
```

Comment:
is it expected that you changed the current AssertConfigMapExists and not created a copy to have both, AssertConfigMapExists and AssertConfigMapDoesNotExist? AssertConfigMapExists might be used in some checkes?

Reply:
This is intentional (see the commit message). We can use GetConfigMap instead and as far as I checked this is not yet used. Alternatively we can add `Assert<resource>MapExists` to all for consistency but I feel like that is redundant.

Reply:
seems it is only used in the infra-operator https://github.com/openstack-k8s-operators/infra-operator/blob/main/tests/functional/dnsmasq_controller_test.go#L226 . need to updated it there.

Reply:
I favor simplicity so lets use GetConfigMap if we want to assert that the CM exists.